src/SmartComponents/HybridInventoryTabs/helpers.test.js

```diff
@@ -121,6 +128,25 @@ describe('getEntities', () => {
     ]);
   });
 
+  test('uses group info from edge API when enforce_edge_groups set to false and isInventoryGroupsEnabled flag is false', async () => {
+    useFeatureFlag.mockReturnValue(false);
+    const { result } = renderHook(() =>
+      useGetEntities(handleRefreshMock, undefined, { rule_id: 'test-rule' })
+    );
+
+    let fetchedResult;
+    await act(async () => {
+      fetchedResult = await result.current(...fetchArguments);
+    });
```

Comment:
Can this be rewritten to just
```suggestion
    fetchedResult = await result.current(...fetchArguments);
```
?